content/quantum_coin_game/quantum_coin_game.ipynb

```diff
@@ -0,0 +1,1071 @@
+    "8. [References](#references)<br>\n",
+    "9. [Quick Exercise](#quick_exercise)<br>\n",
+    "10.[Version Information](#version_information)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "TJDBzM7PFsQN"
+   },
+   "source": [
+    "### **Introduction** <a id=\"introduction\"></a>\n",
+    "\n",
+    "---\n",
+    "\n",
+    "\n",
+    "### What is Quantum Coin Game ? <a id=\"quantum_definition\"></a>\n",
+    "> Quantum Coin Game is one of the fundamental concept of quantum computing, which uses simple implementation of quantum gates or more precisely uses the wierdness of quantum mechanics, to win about 97% of the time, when played against an opponent. Flipping of coin and say heads or tails.\n",
+    "\n",
+    "### Where the concept came from ? <a id=\"concept\"></a>\n",
+    "> The concept of Quantum Coin Game came from the idea of classical coin game which can only show heads and tails. But since the game utilizes the concepts of quantum mechanics, it would be interesting to see what could be the outcome of the whole experiment.\n",
+    "\n",
+    "### What is the main idea of this game ? <a id=\"idea\"></a>\n",
+    "> The main concept of this game is how the quantum computer uses the power of **QUANTUM SUPERPOSITION**, which tells an object can exists in 2 different states at the same time, to win absolutely everytime. The fact is an quantum computer can do superposition already puts it in a higher chance of winning over the classical flip.\n",
+    "> > **NOTE**:To learn more about quantum superposition, link to \"[Qiskit Textbook](https://qiskit.org/textbook/ch-states/representing-qubit-states.html)\" superposition page.\n",
+    "\n",
+    "### What are the rules of this game ? <a id=\"rules\"></a>\n",
+    "> 1. **Quantum Computer** plays a move but it is not revealed to the **Opponent(Human)**.\n",
+    "> 2. **Opponent(Human)** plays a move and it is also not revealed to the **Quantum Computer**.\n",
+    "> 3. Finally **Quantum Computer** plays a move.\n",
+    "> 4. Results are shown. If its heads, then **Quantum Computer** wins. Else, **Opponent(Human)** wins.\n",
+    "> **NOTE**:- \"Playing a move\" refers to \"**Flipping the coin**\".\n",
+    "\n",
+    "**NOTE**: We consider the coin as '**FAIR COIN**'\n",
+    "\n",
+    "\n",
+    "\n",
+    "---\n",
+    "\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "TMsXKAxk6-dt"
+   },
+   "source": [
+    "### **Analogy** <a id=\"analogy\"></a>\n",
+    "\n",
+    "---\n",
+    "\n",
+    "Now that we know what is a quantum coin game, what is it based on and most importantly what are the rules of this game, lets convert the concept of this game in quantum computing terminology.\n",
+    "\n",
+    "* The 'coin' in flipping a coin we referring here is a '**single qubit gate**'.\n",
+    "$$\n",
+    "      |\\psi\\rangle=\\begin{bmatrix}\\alpha \\\\ \\beta\\end{bmatrix}\n",
+    "$$\n",
```

Comment:
I think this equation is not rendering properly.

Reply:
Working for me

Reply:
If it is working in your local system then leave it as it will automatically fixed when pushed to website